subsys/bluetooth/host/hci_core.c

```diff
@@ -2855,6 +2855,11 @@ static void link_key_notify(struct net_buf *buf)
 		break;
 	}
 
+	if (IS_ENABLED(CONFIG_BT_SETTINGS) &&
+		!atomic_test_bit(conn->flags, BT_CONN_BR_NOBOND)) {
```

Comment:
This one should be aligned to the line above, since it is part of the expression in the if statement.

Reply:
I have a question whether it is an idea to mix tabs and spaces?

Reply:
For alignment of split lines, yes. tabs + 0-7 spaces